sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormTrainingClient/FormTrainingClientTests.cs

```diff
@@ -36,14 +37,57 @@ private FormTrainingClient CreateInstrumentedClient()
             return InstrumentClient(client);
         }
 
+        /// <summary>
+        /// Verifies functionality of the <see cref="FormTrainingClient"/> constructors.
+        /// </summary>
+        [Test]
+        public void ConstructorRequiresTheEndpoint()
+        {
+            var tokenCredential = new DefaultAzureCredential();
+            var keyCredential = new AzureKeyCredential("key");
+
+            Assert.Throws<ArgumentNullException>(() => new FormTrainingClient(null, tokenCredential));
+            Assert.Throws<ArgumentNullException>(() => new FormTrainingClient(null, tokenCredential, new FormRecognizerClientOptions()));
+            Assert.Throws<ArgumentNullException>(() => new FormTrainingClient(null, keyCredential));
+            Assert.Throws<ArgumentNullException>(() => new FormTrainingClient(null, keyCredential, new FormRecognizerClientOptions()));
+        }
+
+        /// <summary>
+        /// Verifies functionality of the <see cref="FormTrainingClient"/> constructors.
+        /// </summary>
+        [Test]
+        public void ConstructorRequiresTheTokenCredential()
+        {
+            var endpoint = new Uri("http://localhost");
+
+            Assert.Throws<ArgumentNullException>(() => new FormTrainingClient(endpoint, default(TokenCredential)));
+            Assert.Throws<ArgumentNullException>(() => new FormTrainingClient(endpoint, default(TokenCredential), new FormRecognizerClientOptions()));
+        }
+
+        /// <summary>
+        /// Verifies functionality of the <see cref="FormTrainingClient"/> constructors.
+        /// </summary>
+        [Test]
+        public void ConstructorRequiresTheAzureKeyCredential()
+        {
+            var endpoint = new Uri("http://localhost");
+
+            Assert.Throws<ArgumentNullException>(() => new FormRecognizerClient(endpoint, default(AzureKeyCredential)));
+            Assert.Throws<ArgumentNullException>(() => new FormRecognizerClient(endpoint, default(AzureKeyCredential), new FormRecognizerClientOptions()));
+        }
+
+        /// <summary>
+        /// Verifies functionality of the <see cref="FormTrainingClient"/> constructors.
+        /// </summary>
         [Test]
-        public void CreateClientArgumentValidation()
+        public void ConstructorRequiresTheOptions()
```

Comment:
Separated this in a few methods since we have a lot of things to test (and to keep the same pattern from the `FormRecognizerClientLiveTests`).